nexus/src/app/sagas/instance_create.rs

```diff
@@ -1021,6 +1026,88 @@ async fn sic_delete_instance_record(
     Ok(())
 }
 
+async fn sic_set_boot_disk(
```

Comment:
yeah, either `disk-create` hasn't created the named disk, or `disk-attach` hasn't run yet to actually set the mentioned disk as attached to this instance.

another reason to set the boot disk all the way at the end of instance creation, though, is that setting it any sooner could violate the invariant "`instance.boot_disk_id` references a disk that is attached to the instance"

so, yes, a comment for all this would be good for sure :)